docs/source/usage.rst

```diff
@@ -33,7 +33,7 @@ in Django is declared through the model's Meta class. An example ``models.py``:
 	class SortableBook(models.Model):
 	    title = models.CharField('Title', null=True, blank=True, max_length=255)
 	    my_order = models.PositiveIntegerField(default=0, blank=False, null=False)
-	
```

Comment:
sorry, i was hurry. because doing work with my own project . And was afraid to forget
about pull request.